lang/cs/Org.Apache.REEF.Network/NetworkService/NetworkService.cs

```diff
@@ -65,12 +66,13 @@ public class NetworkService<T> : INetworkService<T>
             IIdentifierFactory idFactory,
             ICodec<T> codec,
             INameClient nameClient,
-            IRemoteManagerFactory remoteManagerFactory)
+            IRemoteManagerFactory remoteManagerFactory,
+            ITcpPortProvider tcpPortProvider)
```

Comment:
My assumption was based on the current implementation. Yes, if TcpPortProvider is shared cross the board, it should be injected into the factory.